src/lib/rng/auto_rng.h

```diff
@@ -35,9 +36,10 @@ class AutoSeeded_RNG : public RandomNumberGenerator
       void add_entropy(const byte in[], size_t len) override
          { m_rng->add_entropy(in, len); }
 
-      AutoSeeded_RNG() : m_rng(RandomNumberGenerator::make_rng()) {}
+      AutoSeeded_RNG(size_t bytes_before_reseed = BOTAN_RNG_MAX_OUTPUT_BEFORE_RESEED);
```

Comment:
The `max number of bytes output before a reseed occurs` parameter is called differently throughout the implementation, here it's `bytes_before_reseed`, in the constructor's definiton it's `max_bytes_before_reseed`, whereas in `Stateful_RNG::Stateful_RNG()` it's `max_output_before_reseed` and in `build-h` it's `BOTAN_RNG_MAX_OUTPUT_BEFORE_RESEED`. I'd like to see a common name for this in all RNGs to avoid confusion.